test/svg/README.md

```diff
@@ -0,0 +1,12 @@
+# SVG File description
+
+Make sure that the viewbox attribute of the file is set properly so that it aligns with the image that it is being tested on.
```

Comment:
What does this even mean? Are you setting a guideline for creating SVG test files? If that's the case add a title and you most probably need to add more rules.

Also, you need to explain more in this doc. You're just listing svg files without actually saying how they should be tested or providing a URL.

Reply:
This is not a guideline. It is to make sure that the SVG file is displayed properly. For example if the content of the svg is outside the image frame of the OSD then even though the SVG is displayed nothing would be visible of the screen. (image h = 100px, w = 100px and svg <circle r='10' cx = '200' cy='200'>). I will be posting the URL in the regression testing doc as I will be adding these cases to that.

Reply:
@rastogi-bhavya As @hongsudt mentioned in our meeting, you should just add some lines explaining these images are to be tested in rbk. You can also add a link from here to the regression test wiki page.